### PR TITLE
adds ko builder image

### PR DESCRIPTION
**Pull request by poy**
This adds a builder image for github.com/google/ko.

### CODE REVIEW

File: ko/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+ENV GO111MODULE=on
+RUN CGO_ENABLED=0 GOOS=linux go get github.com/google/ko/cmd/ko
+
+FROM gcr.io/cloud-builders/kubectl
+
+COPY --from=0 /go/bin/ko /
+COPY ko.bash /builder/ko.bash
+
+# Install Go
```
Comment (ImJasonH) [inline ko/Dockerfile]: I wonder whether you can copy the Go binary out of the builder image... 🤔

Comment (poy) [reply ko/Dockerfile]: That's a cool idea. I'll try that out.

Comment (poy) [reply ko/Dockerfile]: Looks like that works

**Comment by poy** [comment]
@ImJasonH I made the requested update, PTAL.

File: ko/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/ko', '.']
```
Comment (ImJasonH) [inline ko/cloudbuild.yaml]: Can you add a step that sanity checks that ko is available and installed?

```
- name: 'gcr.io/$PROJECT_ID/ko'
  args: ['version']
```

Comment (poy) [reply ko/cloudbuild.yaml]: Done

**Comment by poy** [comment]
@ImJasonH I added the step, PTAL

**Pull request merged**